app/assets/stylesheets/app/dashboard.scss

```diff
@@ -176,7 +195,7 @@
 
   &-extended {
     height: 536px;
```

Comment:
Me parece que en vez de repetir el código de card (que pareciera ser igual el 90% del tiempo) podríamos usar `@extend` ([referencia](https://css-tricks.com/the-extend-concept/))

Reply:
👌